install/kubernetes/helm/istio/values.yaml

```diff
@@ -28,6 +28,16 @@ mixer:
     enabled: true
   telemetry:
     enabled: true
+    averageLatencyThreshold: 75ms
+    # disabled, logonly or enforce
+    loadsheddingMode: enforce
+  adapters:
+    stdio:
+      enabled: false
```

Comment:
i think this is OK... as long as this is still enabled in values-demo-*.yaml.  Is that the case?

Reply:
Please add some comments to the fields as well ( what it means, impact, when should it be on )

Reply:
What is the impact of this to some of our tasks in istio.io?

Reply:
Good question, `-demo` values enable it.

Reply:
checked, https://preliminary.istio.io/docs/tasks/telemetry/metrics-logs/#collecting-new-telemetry-data  The task uses inline config to demonstrate mixer access logs. It does not rely on default config.

Reply:
ok thanks for double checking